freebsd-rc-service.sh

```diff
@@ -0,0 +1,139 @@
+#! /bin/sh -
```

Comment:
Any chance this could be made `/usr/bin/env sh` to align with the shebangs used by other collectors?